MatrixSDK/Data/MXPeekingRoom.m

```diff
@@ -78,12 +80,11 @@ - (MXRoomSummary *)summary
 
 - (void)close
 {
+    [super close];
```

Comment:
It is better to call [super close] at the end. In general clean current object resources first and then clean parent resources.